mulesoft/docs-site-ui
# [W-15921470] trending topics fallback

ref: W-15921470

- add fallback links for when trending topics (from Coveo) aren't returning any results (or bad results)
- update JP site marketing header

### How to test

1. Run `npm run preview-jp`
2. Open http://localhost:8080/jp on your web browser (note: not localhost:8080. I will have to fix that later). You should see initially that the trending topics box shows a bunch of "no titles", and then after a few seconds, they changed into a bunch of real links. These links are hardcoded in a JSON file in case these results are either "no titles" or if there are no results at all
